src/test/java/Assignment16Test.java

```diff
@@ -0,0 +1,19 @@
+import com.assignment.Yes1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.assignment.*;
+
+public class Assignment16Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testAbstractClass() {
```

Comment:
use meaningful name ie ur test name should reflect what operation you are performing ie related to addition or sum